pkg/v2/object-propagation/README.md

```diff
@@ -0,0 +1,31 @@
+### Object Propagation Controller
+
+This controller source objects into the target namespaces. 
+
+Each set of source objects is specified using:
+- apiVersion - required
+- kind - required
+- namespace - required
+- label selector - may be empty ("")
+
+The target namespace is specified by a label selector, which may be empty ("").   
+
+The controller reads configuration provided via `--input` CLI parameter (default: `/dev/stdin`). 
+Example input:
+
+```yaml
+- source:
+    apiVersion: v1
+    kind: ConfigMap
+    namespace: tanzu-system
+    selector: 'run.tanzu.vmware.com/propagated'
```

Comment:
There's an example that shows it's a string. Also, this config is not Kubernetes API. My reasoning for this format was: what's the simplest and shortest form to convey a list of source-and-target configurations? 

Initially, it was just a bunch of CLI params. It's a YAML file now, because it's not easy to express a list of key-value maps as CLI params (not to me at least). We need a list because we need to propagate not just ClusterClass, but also a bunch of related resources, like infra-ClusterTemplate, infra-MachineTemplate, etc.

Reply:
In that case do you think it  would  be more readable to change `selector` to `labelSelector`. It would convey that selector is based on labels.

Reply:
Renamed